tools/test_history.py

```diff
@@ -206,8 +203,10 @@ def description() -> str:
     2021-02-10 10:09:10Z 2e35fe95 (no reports in S3)
     2021-02-10 10:09:07Z ff73be7e (no reports in S3)
     2021-02-10 10:05:39Z 74082f0d (no reports in S3)
-    2021-02-10 07:42:29Z 0620c96f pytorch_linux_xenial_py3_6_gcc5_4_test 0.414s (1 S3 reports omitted)
-    2021-02-10 07:42:29Z 0620c96f pytorch_linux_xenial_py3_6_gcc7_test 0.377s (1 S3 reports omitted)
+    2021-02-10 07:42:29Z 0620c96f pytorch_linux_xenial_py3_6_gcc5_4_test 0.414s
+    2021-02-10 07:42:29Z 0620c96f pytorch_linux_xenial_py3_6_gcc5_4_test 0.476s
```

Comment:
sort of confusing that these are the same timestamp.. (i assume this is the commit time?) do we have the s3 file upload time we can display here?

Reply:
yes, this is the commit time; I wasn't sure it would be a great idea to always use the S3 file upload time even when there aren't re-runs (and also, do our existing utility functions provide access to those timestamps?)